us.materialio.Materialious.metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id type="desktop-application">us.materialio.Materialious</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Materialious</name>
+  <summary>Materialious is a privacy respecting frontend for YouTube built ontop of Invidious.</summary>
+  <description>
+    <p><i>Materialious</i> is a client for Invidious using material design. It lets you browse and watch videos, manage playlists and vist channels.</p>
+    <p>
+        <b>Features:</b>
```

Comment:
<b> isn't a valid tag, it only supports a subset of HTML https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#description